config/test.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+module.exports = {
+  FOO: 'bar'
+};
```

Comment:
since you have another `module.exports` line down below, this first one is effective being overwritten so you can just remove it. same for `config/development.js`